### DISCUSSION TITLE
Shorten URL at alert messages

### DISCUSSION THREAD

**Discussion by vbulbul** [grafana/grafana#71716] [Alerting]
There is an option on “Share Panel” for shorten URL. However on alert rules URL on telegram messages are very long. Can Shorten URL option be added to alert messages ? 
![image](https://github.com/grafana/grafana/assets/39090711/8beedc10-df16-4f91-8c1d-a34bbc6c72f1)
![image](https://github.com/grafana/grafana/assets/39090711/8ded38ac-a668-489b-a9d1-95910bf5aede)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82616. Feel free to continue the discussion around this there. Thank you!
